### PR TITLE
First draft of user interaction controls

### LINKED ISSUE #16: Create an article on using a control to enable interactive experiments
Based on a recent conversation in the Chaos Toolkit community slack, this will be a walkthrough article with source code to show how to create a control that enables several styles of interaction with the user through the Chaos Toolkit. Styles planned include:

* Simple "Click to continue..." style interaction
* Yes/No to running an activity style interaction
* This is a Dangerous Activity, skip or proceed enhancement to the previous style of interaction.

### PR DESCRIPTION
**Pull request by russmiles**
Closes #16

Signed-off-by: Russ Miles <russ@russmiles.com>

### CODE REVIEW

File: interactive-controls/README.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+In this article you're going to walk through how to create a new [Chaos Toolkit][chaostoolkit]
+[Control][control] to enable various styles of user interaction where needed when an 
+automated chaos experiment is being executed.
+
+[chaostoolkit]: https://chaostoolkit.org/
+[control]: https://docs.chaostoolkit.org/reference/extending/create-control-extension/
+
+> ***NOTE:*** This article was originally inspired by a conversation with 
+> [Simon Lindroos][simon] on the [Chaos Toolkit Community Slack][slack], and also relates to the 
+> [older issue][dangerous-issue] originally raisded on the [Chaos Toolkit][chaostoolkit]
```
Comment (Lawouach) [inline interactive-controls/README.md]: typo on raised :)

File: interactive-controls/README.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+without user interaction. This is still the most desirable strategy, as it makes it 
+possible to schedule experiments to be executed automatically without having to 
+noitify the team to make people available. Two cases where this is useful are 
+when your chaos experiments are being used as chaos tests in a CI/CD system, or 
+as part of the [Chaos Platform's][chaosplatform] scheduling capability.
+
+[chaosplatform]: https://github.com/chaostoolkit/chaosplatform
+
+However there are a number of cases when parts of, or all of, an experiment should 
+involve user interaction. As [Simon Lindroos][simon] pointed out in the original
```
Comment (Lawouach) [inline interactive-controls/README.md]: Another interesting is in a gameday. The operator runs the gameday step by step to give time the team to explore the system before the next thing happens :)

File: interactive-controls/README.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+interaction styles can be implemented using the powerful [Chaos Toolkit Control][control] 
+extension point:
+
+* Simple "Click to continue..." style interaction.
+* "Yes/No" to running an activity style interaction.
+* "This is a Dangerous Activity, skip or proceed" enhancement to the previous style of interaction Shows one strategy to meeting the need specified in [this issue][dangerous-issue].
+
+## Introducing the three new Chaos Toolkit Extensions
+
+This article comes with three extension for each of the above user interaction styles.
```
Comment (Lawouach) [inline interactive-controls/README.md]: extensions ;)

**Review by Lawouach — APPROVED** [review]
LGreatTM :)

File: interactive-controls/README.md (hunk 2 above)
Comment (russmiles) [reply interactive-controls/README.md]: Love that! Added it.

**Pull request merged**